source/MRTest/CMakeLists.txt

```diff
@@ -4,7 +4,11 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 project (MRTest CXX)
 
-add_executable(${PROJECT_NAME} MRTestApp.cpp MRTriangleIntersection.cpp)
+add_executable(${PROJECT_NAME}
+    MRTestApp.cpp
+    MRTriangleIntersection.cpp
+    MRZlib.cpp
```

Comment:
better include all .cpp's from the folder as in other projects

Reply:
Fixed.